cmd/genesis/values.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"errors"
+)
+
+const (
+	DefaultSmartContractsOwner = "0x259be75d96876f2ada3d202722523e9cd4dd917d"
```

Comment:
This value is derived from the [network code](https://github.com/kowala-tech/kcoin/blob/dev/contracts/network/election.go#L20), and I suggest that it only be defined only in election.go. 

In order to do that, `mapChainIDToAddr` would have to be exported and this const would have to refer to it (like `network.MapChainIDToAddr[params.TestnetChainConfig.ChainID.Uint64()]`, which is slightly ugly) but then at least the defaults will be always be up to date with the core code.

Reply:
Great! Will do this.

Reply:
I understand that this should be defined based on the network selection, main or test